docs/zh-CN/docs/sql-manual/sql-functions/aggregate-functions/grouping.md

```diff
@@ -25,25 +25,82 @@ under the License.
 -->
 
 ## GROUPING
-### description
-#### Syntax
 
-`GROUPING(expr)`
+用在含有 CUBE、ROLLUP 或 GROUPING SETS 的SQL语句中，用于表示进行 CUBE、ROLLUP 或 GROUPING SETS 操作的列是否汇总。当结果集中的数据行是 CUBE、ROLLUP 或 GROUPING SETS 操作产生的汇总结果时，该函数返回 1，否则返回 0。GROUPING 函数可以在 `SELECT`、`HAVING` 和 `ORDER BY` 子句当中使用。
```

Comment:
add space after and before English letters in CN docs

Reply:
I have made corrections.